protocol/v1/qbft/instance/change_round_test.go

```diff
@@ -248,14 +249,7 @@ func TestValidateChangeRoundMessage(t *testing.T) {
 		operatorIds[1]: secretKeys[operatorIds[1]],
 	}
 
-	threeSigners := map[spectypes.OperatorID]*bls.SecretKey{
-		operatorIds[0]: secretKeys[operatorIds[0]],
-		operatorIds[1]: secretKeys[operatorIds[1]],
-		operatorIds[2]: secretKeys[operatorIds[2]],
-	}
-
 	consensusMessageSignatureByTwo := aggregateSign(t, twoSigners, consensusMessage)
-	consensusMessageSignatureByThree := aggregateSign(t, threeSigners, consensusMessage)
```

Comment:
why removing 3 signer test?

Reply:
@nivBlox Because spec uses an array of prepare messages instead of one that holds an aggregated message. The `valid justification 1` was using `consensusMessageSignatureByThree`, but spec validation requires having 3 justifications with single signatures instead of an aggregated one, so `consensusMessageSignatureByThree` became unused. However, `twoSigners` is still used for the `invalid justification sig` case.